sklearn/compose/_column_transformer.py

```diff
@@ -721,20 +721,36 @@ def _call_func_on_transformers(
         Return value (transformers and/or transformed X data) depends
         on the passed function.
         """
+        if func is _fit_transform_one:
+            fitted = False
+        else:  # func is _transform_one
+            fitted = True
+
```

Comment:
There are two reasons to keep `fitted` as a separate parameter: (i) explicit is better than implicit; (ii) potentially when `partial_fit` is implemented we'll need `fitted=False` for the first call and `fitted=True` for subsequent calls.

Reply:
this was as a result of a previous review, and I rather keep it as is now.